BUILD.gn

```diff
@@ -218,7 +218,10 @@ group("create_dist") {
   }
 
   if (is_win) {
-    deps += [ "build/win:create_signed_installer" ]
+    deps += [
+      "build/win:create_signed_installer",
+      "components/policy:pack_policy_templates",
```

Comment:
these relative GN deps are bad. From the first glance it's unclear if they refers to a Brave target or an upstream target. 
GN style guide is pretty explicit about this:
> Other deps should always use fully-qualified path names unless relative ones are required for some reason.

https://gn.googlesource.com/gn/+/refs/heads/main/docs/style_guide.md#deps

Reply:
Resolved in 226f465587fae0e92eed50ae037b5afa707b9b79.